extensions/sql-database-projects/src/test/testUtils.ts

```diff
@@ -34,6 +35,10 @@ export async function createTestSqlProjFile(contents: string, folderPath?: strin
 }
 
 export async function createTestProject(contents: string, folderPath?: string): Promise<Project> {
+	const macroDict: Record<string, string> = {
```

Comment:
What's this for?

Reply:
this is to fix the unit test failures, the test project does not have DSP information set previously, the new code I added requires it to be available otherwise the project.getTargetPlatform() will throw error 'invalid DSP.'.